packages/ts-moose-lib/src/typeConvert.ts

```diff
@@ -53,7 +54,8 @@ const tsTypeToDataType = (
                     typeName,
                   )[1],
                 }
-              : nonNull.isClassOrInterface()
+              : nonNull.isClassOrInterface() ||
+                  (nonNull.flags & TypeFlags.Object) !== 0
```

Comment:
How does `TypeFlags.Object` work?

Reply:
they're bitwise flags
the API provides those checking as a method for other type of types, but not the object type